### ISSUE TITLE
Behaviour of the Categories

### ISSUE THREAD

**Issue by martinschaible** [blueriver/MuraCMS#788]
Hello,

The Admin UI let a you add any folder and any page to a category.

This is the test case:

Home - Folder 1 -> Page 1
Home - Folder 1 -> Folder 2

.:.Linking Folder 1 to a category will increment the # of members in the choosen category.
.:. Linking Folder 2 to a category will NOT increment the # of members in the choosen category.
.:. Linking Page 1 to a category will NOT increment the # of members in the choosen category.

In other words:
.:. The Admin UI lets you do stuff which won't work.
.:. Categories can not be linked to pages
.:. Categories can only be linked to folders on level 1.

Solutions:
1. Enhance the UI by disabling "Categorization" on positions where it's not supported.
2. Allow to link anything to a category.

**Comment by stevewithington** [comment]
I believe you were referring to the baked-in "Category Summary" display object and the number of content items displayed. This is a "folder" display object. The default behavior of this is tied directly to the selected folder and its immediate children.

However, if you use the Content Collections/Local Indexes area, you can easily create a feed of any categorized content.
